### ISSUE TITLE
1.18.1 Sketchy console output  for MobEntityMixin.getXpToDropMixin()

### ISSUE THREAD

**Issue by davidalb97** [Globox1997/RpgDifficulty#20]
I was looking for errors on the console and found the following console output:
```
/*****************************************************************************************************************/
/*         Target Class : net.minecraft.class_1308                                                               */
/*        Target Method : method_6110                                                                            */
/*        Callback Name : getXpToDropMixin                                                                       */
/*         Capture Type : int                                                                                    */
/*          Instruction : [101] InsnNode IRETURN                                                                 */
/*****************************************************************************************************************/
/*           Match mode : IMPLICIT (match single) - VALID (exactly 1 match)                                      */
/*        Match ordinal : any                                                                                    */
/*          Match index : any                                                                                    */
/*        Match name(s) : any                                                                                    */
/*            Args only : false                                                                                  */
/*****************************************************************************************************************/
/* INDEX  ORDINAL                            TYPE  NAME                                                CANDIDATE */
/* [  1]    [  0]                      class_1657  arg0                                                -         */
/* [  2]    [  0]                             int  $$1                                                 YES       */
/* [  3]                                        -                                                                */
/*****************************************************************************************************************/
```

Is this something to be concerned about?

Affected mixin:
```
net.rpgdifficulty.mixin.MobEntityMixin.getXpToDropMixin()
```

**MC Version: 1.18.1**

**Mod list:**
- rpgdifficulty 1.0.11
- fabricloader 0.13.3
- fabric 0.46.6+1.18
- cloth-config 6.2.57

**Comment by Globox1997** [comment]
No it indicates not a problem but makes the extra xp not working 
I forgot or didn't notice a set boolean but it will be fixed in the next update (which will be 1.18.2)

**Comment by davidalb97** [comment]
Too bad this fix won't be backported to 1.18.1, most mods are still stuck on that version. 
I'm making a modpack currently on 1.18.1, do you recommend using an older version of your mod as a workaround?

**Comment by Globox1997** [comment]
Yes I recommend to use an older version :)
